src/backgroundblurprocessor/BackgroundBlurProcessorProvided.ts

```diff
@@ -139,15 +146,27 @@ export default class BackgroundBlurProcessorProvided implements BackgroundBlurPr
 
     this.spec = spec;
     this.logger = options.logger;
+    this.videoFramesPerFilterUpdate = 1;
 
     this.setBlurStrength(options.blurStrength);
     this.delegate = new BackgroundBlurVideoFrameProcessorDelegate();
     this.frameCounter = new BackgroundFilterFrameCounter(
       this.delegate,
       options.reportingPeriodMillis,
+      options.filterCPUUtilization,
       this.logger
     );
 
+    this.cpuMonitor = new BackgroundFilterMonitor(options.reportingPeriodMillis * 5, {
```

Comment:
why the *5?

Reply:
I wanted to leave some time to adapt. By default the reporting period is 1 second. I think 5 seconds is a good amount of time to settle in to segmentation frequency change and the reevaluate the performance.

Reply:
Ok this can get tricky if somebody sets the reporting period to 5s or 0.5s. Can we just set it to a flat 5s value?